File: scratch/list_users.py
import sqlite3

db_path = r'c:\Users\JULIAN\Desktop\GravyLocal2.0\pb_data\data.db'
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

cursor.execute("PRAGMA table_info(users)")
cols = cursor.fetchall()
print("Columns of users table:")
for col in cols:
    print(col)

cursor.execute("SELECT id, email, role, active FROM users")
rows = cursor.fetchall()
print("\nUsers in DB:")
for r in rows:
    print(r)

conn.close()

File: scratch/list_users.js
const sqlite3 = require('sqlite3').verbose();
const dbPath = 'c:/Users/JULIAN/Desktop/GravyLocal2.0/pb_data/data.db';

const db = new sqlite3.Database(dbPath, sqlite3.OPEN_READONLY);

db.all("SELECT id, email, role, active FROM users", [], (err, rows) => {
  if (err) {
    console.error(err);
    return;
  }
  console.log("=== USERS IN LOCAL DB ===");
  console.log(JSON.stringify(rows, null, 2));
});
